src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationMfaStatusController.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.professionalapi.controller.external;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.professionalapi.controller.SuperController;
+import uk.gov.hmcts.reform.professionalapi.controller.response.MfaStatusResponse;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RequestMapping(
+        path = "/refdata/external/v1/organisations/mfa"
+)
+@RestController
+public class OrganisationMfaStatusController extends SuperController {
+
+    @ApiOperation(
+            value = "Retrieves the mfa status of the organisation a user belongs to"
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "The mfa status of the organisation the user belongs to",
+                    response = MfaStatusResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request has been provided"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "The requested user does not exist"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
+            )
+    })
+
+    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<MfaStatusResponse> retrieveMfaStatusByUserId(
+            @RequestParam(value = "user_id", required = true) String userId) {
+
+        MfaStatusResponse mfaStatusResponse = mfaStatusService.findMfaStatusByUserId(userId);
+
+        return ResponseEntity
+                .status(200)
```

Comment:
I think we should not return `200` here but instead from the `mfaStatusService` method. Otherwise we will always return 200 success when it may not be successful - we had similar issue to another API before

Reply:
Good catch Adil.